docs/ug/configFiles.md

```diff
@@ -93,10 +93,20 @@ e.g.: `example.java` in `example-repo` can either be in the `test` group or the
 
 ## `report-config.json`
 
-You can optionally use `report-config.json` to customize report generation by providing the following information. ([example](report-config.csv))
+You can optionally use `report-config.json` to customize report generation by providing the following information. ([example](report-config.json))
 
 **Fields to provide**:
 * `title`: Title of the generated report, which is also the title of the deployed dashboard. Default: "RepoSense Report"
+* `sinceDate`: The first day of the period to be analyzed, in the format `DD/MM/YYYY`. Default: One month before the current date.
+* `untilDate`: The last day of the period to be analyzed, in the format `DD/MM/YYYY`. Default: Current date.
+* `period`: The period of the analysis window, in the format `nd` (for `n` days) or `nw` (for `n` weeks). It is used to calculate end date if only start date is specified, or calculate end date if only start date is specified.
+
+<box type="info" seamless>
+
+* `sinceDate`, `untilDate` and `period` cannot be declared together and will be ignored if all 3 configuration settings are present in the report configuration.
```

Comment:
What's the reason for ignoring all 3? Why not just take since and until in this case, assuming nothing provided in CLI arguments. Don't do changes as of now, just explain the intuition behind this logic for now.

Reply:
I followed the logic originally present for CLI arguments, whereby if all 3 arguments are provided we will throw an exception. In this case, because it is in report config, I chose to ignore instead of throwing an exception.